src/main/java/com/assignment/Assignment4.java

```diff
@@ -0,0 +1,8 @@
+package com.assignment;
+
+public class Assignment4 {
+
+    public String returnSubString(String originalString,String subString) {
```

Comment:
The naming convention for a method should be :
getMethod-name -> if its returning something
setMethod-name -> if its setting something